lib/node_modules/@stdlib/stats/base/varianceyc/test/test.varianceyc.js

```diff
@@ -93,6 +131,25 @@ tape( 'if provided an `N` parameter less than or equal to `0`, the function retu
 	t.end();
 });
 
+tape( 'if provided an `N` parameter less than or equal to `0`, the function returns `NaN (accessor)`', function test( t ) {
+	var x;
+	var v;
+
+	x = toAccessorArray([ 1.0, -2.0, -4.0, 5.0, 3.0 ]);
+
+	v = varianceyc( 0, 1, x, 1 );
+	t.strictEqual( isnan( v ), true, 'returns expected value' );
+
+	v = varianceyc( -1, 1, x, 1 );
+	t.strictEqual( isnan( v ), true, 'returns expected value' );
+
+	x = toAccessorArray([ 10.0 ]);
+	v = varianceyc( 0, 0, x, 1 );
+	t.strictEqual( isnan( v ), true, 'returns expected value when N = 0 with a single value' );
```

Comment:
```suggestion
	t.strictEqual( isnan( v ), true, 'returns expected value' );
```

Same comments here and elsewhere.